models/image.js

```diff
@@ -0,0 +1,7 @@
+const mongoose = require('mongoose');
+
+const imageSchema = new mongoose.Schema({
+  urls: [String],
```

Comment:
On diagram image model got id_room and a unique url ( not an array ) is this normal ?